activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -773,7 +773,7 @@ def configure_connection
           sql_mode_assignment = "@@SESSION.sql_mode = #{sql_mode}, " if sql_mode
 
           # NAMES does not have an equals sign, see
-          # http://dev.mysql.com/doc/refman/5.7/en/set-statement.html#id944430
+          # https://dev.mysql.com/doc/refman/5.7/en/set-statement.html
```

Comment:
@kamipo Thanks for advising a correct link. I've followed your advice, and fixed it.